code/game/objects/items/devices/organ_module/active/armguns.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/organ_module/active/simple/armsmg
 	name = "embedded SMG"
-	desc = "An SMG designed to be inserted into an arm. Gives you a nice advantage in a firefight"
+	desc = "A SMG designed to be inserted into an arm. Gives you a nice advantage in a firefight"
 	verb_name = "Deploy embedded SMG"
```

Comment:
`An` is the correct indefinite article here, although the sentence should end with a period.

Reply:
Why is an correct?

Reply:
It depends on the following word's sound. If it begins with a consonant sound (ie cucumber, rabbit, pen) it gets "a".
A cucumber, a rabbit, a pen.

But if it begins with a vowel sound (apricot, antelope, idea), it gets "an".

The reason `An SMG` is correct, is because SMG begins with the sound "es".
(It's one of those rules that most native speakers don't really learn, just because it's intuitive for them.)

Reply:
(Also, I just noticed the ending punctuation is missing from this line.